static/canvas.js

```diff
@@ -200,45 +187,48 @@ class Canvas {
 
```

Comment:
@allenywang I'm not sure what you mean by this. Which line are you talking about?